app/views/ops/_analytics_details_tab.html.haml

```diff
@@ -1,8 +1,9 @@
 - if @sb[:active_tab] == "analytics_details"
   - if @sb[:analytics_rpt]
     %fieldset
-      %h3= @sb[:analytics_rpt].title
-      - if !@sb[:analytics_rpt].table.nil? || @sb[:analytics_rpt].table.data.empty?
+      %h3
+        = @sb[:analytics_rpt].title
+      - if !@sb[:analytics_rpt].table.present? || !@sb[:analytics_rpt].table.data.empty?
```

Comment:
@romanblanco
**So, the old code reads to me like this:**
`if the table is not nil (present) OR the table's data is empty` . . . which would be TRUE for the first part if there is a table (empty or not), so probably always worked because there is usually a table and it's not normally empty and it never checked the second part, so will be TRUE overall
**The new code reads like this:**
`if the table is not present OR the table's data is NOT empty` . . . which will normally be FALSE for the first part and TRUE for the second part, so will be TRUE overall
**Should read like this:**
`if the table is present AND the table's data is NOT empty` (note the change from OR to AND) . . . which will make sure we 1) have a table and 2) it's not empty

Logically, the last one seems right to me.  If that causes an exception, let me know so we can see what's causing it as, from what I understand of the ruport structure we are using, this should work.